packages/browser/scripts/checkbrowsers.js

```diff
@@ -0,0 +1,132 @@
+// Script which checks all browsers in test/integration/browser.js against supported BrowserStack browsers
+// Meant to be run manually, by running `yarn test:integration:checkbrowsers` from the command line
+
+const btoa = require('btoa');
+const fetch = require('node-fetch');
+const localConfigs = require('../test/integration/browsers.js');
+
+const browserstackUsername = process.env.BROWSERSTACK_USERNAME;
+const browserstackAccessKey = process.env.BROWSERSTACK_ACCESS_KEY;
+
+const hasCreds = () => {
+  return browserstackUsername !== undefined && browserstackAccessKey !== undefined;
+};
+
+const fetchCurrentData = (username, key) => {
+  const authKey = btoa(`${username}:${key}`);
+
+  return fetch('https://api.browserstack.com/5/browsers?flat=true', {
+    headers: {
+      Authorization: `Basic ${authKey}`,
+    },
+  }).then(response => {
+    if (response.status >= 200 && response.status < 300) {
+      return response.json();
+    } else {
+      throw new Error(`Unable to fetch data. Status: ${response.status} ${response.statusText}`);
+    }
+  });
+};
+
+const isMatchingEntry = (key, localConfig, bsConfig) => {
+  let localValue = localConfig[key];
+  let bsValue = bsConfig[key];
+
+  // all values are either null, undefined, or strings, so this should only catch strings
+  if (localValue) {
+    localValue = localValue.toLowerCase();
+  }
+  if (bsValue) {
+    bsValue = bsValue.toLowerCase();
+  }
+
+  if (localValue === bsValue) {
+    return true;
+  }
+  if (key === 'browser_version' && localValue === 'latest') {
+    return true;
+  }
+
+  return false;
+};
+
+const isMatchingConfig = (localConfig, bsConfig) => {
+  const checkKeys = ['os', 'os_version', 'browser', 'device', 'browser_version'];
+
+  for (const key of checkKeys) {
```

Comment:
It can also be written as:

```js
if (checkKeys.some(key => !isMatchingEntry(key, localConfig, bsConfig)) {
  return false;
}
```

but I like `for` more anyway

Reply:
Oh, right, of course - just like

```
const isSupported = (localConfig, supportedConfigs) => {
  return supportedConfigs.some(supportedConfig => isMatchingConfig(localConfig, supportedConfig));
};
```

I actually like the `some` way better, because it's almost English: "If some key leads to a non-matching entry, return false."

Thanks!